### DISCUSSION TITLE
Theming in v2

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#5597] [Help]
Even though the theming ability in v3 will be improved drastically, I would like to make everything a little bit more edgy in v2. According to the docs (https://filamentphp.com/docs/2.x/admin/appearance#building-themes) right now it is only possible to change the colors and the CSS. However it is not possible to change the tailwind classes. Is this correct?

**Comment by pxlrbt** [comment]
Sure. You can adjust your tailwind config however you want. If you want to overwrite the border radius, go ahed.

**Reply by zepfietje** [reply]
To add, you can actually add your own Tailwind CSS classes to elements by targeting `filament-` CSS classes on elements. Nothing really changed with respect to that in v3 compared to v2 (except we aim to add more `filament-` CSS hooks).

**Comment by danharrin** [comment]
The only thing that changes in v3 is that colours and fonts can be customised without a Tailwind build
